.github/workflows/cd.yaml

```diff
@@ -19,11 +20,11 @@ jobs:
             - name: Setup Node
               uses: actions/setup-node@v1
               with:
-                  node-version: 14
+                  node-version: 16
 
             - name: Fetch AWS Credentials
               run: |
-                  export AWS_ROLE_ARN=${{ secrets.PROD_US_EAST_1_ROLE }}
```

Comment:
configure-aws-credentials got native support for this flow so will be good to migrate to it at some point

Reply:
I tried configure-aws-credentials when I first set this up, but I ran into issues (i.e., the deployment was failing and difficult to debug -- see https://github.com/aws-observability/aws-rum-web/pull/4).

Maybe its worth revisiting now that it's a few months more mature, but this `curl` method works fine so I'm not sure I want to put effort into it.